src/app/content/content.browserspec.ts

```diff
@@ -125,21 +129,17 @@ const clickTocLink = (href: string) => page.evaluate(async(href) => {
 }, href);
 
 const getSelectedTocSection = () => page.evaluate(() => {
-  const tocHeader = document && Array.from(document.querySelectorAll('h2'))
-    .find((node) => node.textContent === 'Table of Contents');
-
-  const li = tocHeader && tocHeader.parentElement
-    && tocHeader.parentElement.querySelector('li[aria-label="Current Page"]');
+  const toc = document && document.querySelector('[aria-label="Table of Contents"]');
 
+  const li = toc && toc.querySelector('li[aria-label="Current Page"]');
```

Comment:
are there more `[aria-label="Current Page"]` elements? Or is this just clarifying that the element should be an `li`?

Reply:
the li here is irrelevant, its just for readability, but it might be brittle, should we remove it?